### PR TITLE
Specifying JSON modules

### PR DESCRIPTION
**Pull request by littledan**
This PR also fixes #52

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -70,26 +70,279 @@ <h1>Runtime Semantics: Evaluation</h1>
...
+    </emu-alg>
+    <!-- TODO: Could we apply the new spec closure concept here rather than painstakingly passing _defaultExport_ through [[HostDefined]]? -->
+  </emu-clause>
+
+  <emu-clause id="sec-parse-json-module" aoid="ParseJSONModule">
+    <h1>ParseJSONModule ( _source_ )</h1>
+    <p>The abstract operation ParseJSONModule takes a single argument _source_ which is a String representing the contents of a module.</p>
+
+    <emu-alg>
+      1. Let _json_ be ? Call(%JSONParse%, *undefined*, « _source_ »).  <!-- TODO: Refactor JSONParse into an abstract algorithm -->
```
Comment (Jack-Works) [inline spec.html:334]: Should the JSON be frozen? What if a developer accidentally call the `json.arr.sort()` then the file is corrupted

Comment (littledan) [reply spec.html:334]: Let's discuss this in a separate issue: https://github.com/tc39/proposal-module-attributes/issues/54

File: spec.html [hunk 2]
```diff
@@ -70,26 +70,279 @@ <h1>Runtime Semantics: Evaluation</h1>
...
           1. IfAbruptRejectPromise(_specifierString_, _promiseCapability_).
-          1. <ins>Let _attrRef_ be the result of evaluating the second |AssignmentExpression|.</ins>
-          1. <ins>Let _attr_ be ? GetValue(_attrRef_).</ins>
-          1. <ins>If Type(_attr_) is not String,</ins>
-            1. <ins>Call(_promiseCapability.[[Reject]], *undefined*, a new TypeError exception).</ins>
-          1. <ins>Otherwise,</ins>
-            1. Perform ! HostImportModuleDynamically(_referencingScriptOrModule_, _specifierString_, <ins>_attr_,</ins> _promiseCapability_).
+          1. <ins>Let _argRef_ be the result of evaluating the second |AssignmentExpression|.</ins>
+          1. <ins>Let _arg_ be ? GetValue(_argRef_).</ins>
+          1. <ins>If _arg_ is *undefined*, let _attributes_ be *undefined*.</ins>
```
Comment (JLHwung) [inline spec.html:75]: Note that this PR also fixes #52 .

Comment (littledan) [reply spec.html:75]: Good point, added to the commit message.

**Comment by littledan** [comment]
I'm going to land this PR; review comments still welcome.

**Pull request merged**